src/App.js

```diff
@@ -9,7 +11,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
I guess you can use spread here

Reply:
Taras is right. You can just pass the `article object to the `props` object
`<Article {...article } />